source/internal.d.ts

```diff
@@ -76,6 +77,16 @@ Matches non-recursive types.
 */
 export type NonRecursiveType = BuiltIns | Function | (new (...args: any[]) => unknown);
 
+/**
+Returns a boolean for whether the given type is a normal key-value object.
```

Comment:
It has to be more specific what a "normal key-value object" is exactly. Plain object?